webapp/src/components/MarketplacePage/Publication/Publication.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react'
+import { Header, Card, Button, Icon } from 'semantic-ui-react'
+import { Link } from 'react-router-dom'
+import { publicationType } from 'components/types'
+import { txUtils } from 'decentraland-commons'
+import EtherscanLink from 'components/EtherscanLink'
+import numeral from 'numeral'
+import moment from 'moment'
+import './Publication.css'
```

Comment:
I know this is nitpicking, but can we keep the import order more or less simliar to `MarketplacePage.js`s ? (which is how most components order the imports

I think that instead of just pointing it out here I should add a PR to the standards, but in my head this makes sense:

```javascript
import React from 'react'
// React related imports, like react-router-dom or prop-types
// general imports, like decentraland-commons or lodash.debounce

// Components, semantic-ui-react first just to distinguish it
// Component/Types

// Lib or Utils

// CSS
```

I don't intend to always keep _that_ template, but I do think that grouping imports by "category" can be useful for taking a quick look.